### PR TITLE
Add Splashpage 

### LINKED ISSUE #5: Build Splash Screen
## User story

SplashScreen should welcome the user. Should last 1,5 Sec.

## ToDos

-[ ]
-[ ]
-[ ]
-[ ]

### PR DESCRIPTION
**Pull request by Phil9219**

### CODE REVIEW

File: client/.storybook/preview.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+export const decorators = [
+  (Story) => (
+    <>
+      <GlobalStyle />
+      <Story />
+    </>
+  ),
+];
+export const parametesr = {
```
Comment (kschmitz123) [inline client/.storybook/preview.js:12]: there is a typo:

```suggestion
export const parameters = {
```

File: client/src/App.js [hunk 2]
```diff
@@ -1,15 +1,12 @@
 import "./App.css";
-import Page from "../src/components/Page/Page";
+
+import SplashPage from "../src/components/Page/SplashPage";
```
Comment (kschmitz123) [inline client/src/App.js:1]: i think this shorter import should also work:

```suggestion
import SplashPage from "./components/Page/SplashPage";
```

**Review by kschmitz123 — COMMENTED** [review]
lgtm 👍

File: client/src/components/Page/SplashPage.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Splashlogo from "../../assets/logo_JRK.svg";
```
Comment (kschmitz123) [inline client/src/components/Page/SplashPage.js:2]: you could probably shorten the import link here as well

File: client/src/GlobalStyle.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { createGlobalStyle } from "styled-components";
```
Comment (lmachens) [inline client/src/GlobalStyle.js:1]: You will get better logs and developer experience if you use [babel-macro](https://styled-components.com/docs/tooling#babel-macro):

```suggestion
import { createGlobalStyle } from "styled-components/macro";
```

File: client/src/components/Header/Header.js [hunk 5]
```diff
@@ -1,63 +1,22 @@
...
+  img {
+    height: 55px;
+    padding: 0 2rem;
+  }
+`;
 
-const Header = ({ user, onLogin, onLogout, onCreateAccount }) => (
-  <header>
+const Header = () => (
+  <AppHeader>
```
Comment (lmachens) [inline client/src/components/Header/Header.js:15]: It sounds wrong that the `Header` contains a `AppHeader`. I would swap the names and call this file `AppHeader`.

**Review by lmachens — COMMENTED** [review]
gj

File: client/src/GlobalStyle.js (hunk 4 above)
Comment (Phil9219) [reply client/src/GlobalStyle.js:1]: Thanks

File: client/src/components/Header/Header.js (hunk 5 above)
Comment (Phil9219) [reply client/src/components/Header/Header.js:15]: Done. Thanks

**Pull request merged**